### DISCUSSION TITLE
Export to Quizlet/Anki

### DISCUSSION THREAD

**Discussion by swoday07** [st3v3nmw/obsidian-spaced-repetition#713] [Q&A]
Hi, 
I was wondering if there was a way to export all flashcards to a format supported by Quizlet/Anki? I'm a language learner, and exporting the cards to Quizlet would let me hear the audio for the new vocabulary I'm learning.
I have lots of atomic notes, and it would take a while to export all of them one by one.

Alternatively, is there a way to export all notes that have "::" in them? I could then use dataview to create a table containing both sides of my flashcards, and then export that to Anki/Quizlet.

**Comment by rdpat1711** [comment]
I am looking for the solution since last 10 hours, and the one that i found is unfortunately not working. However you can try out Obsidian to Anki plugin. It works for a lot of individuals

**Comment by hurryclear** [comment]
I hope this plugin can also add function that enable me to export flashcards that I made with space repetition style to anki. Yes, there are some plugins that sync obsidian notes to anki, but the note taking style are too various. I want to stick to the style this plugin used. 
Also I feel there is need for me to use both space repetition flashcard and anki together.
It would be great if this workflow can be achieved!
Anyone has solution already?!
